process/smartContract/process.go

```diff
@@ -444,14 +457,58 @@ func (sc *scProcessor) executeSmartContractCall(
 	return vmOutput, nil
 }
 
+func (sc *scProcessor) isSCROnlyInformational(scr data.TransactionHandler) bool {
+	if scr.GetValue().Cmp(zero) > 0 {
+		return false
+	}
+
+	function, _, err := sc.argsParser.ParseCallData(string(scr.GetData()))
+	if err != nil {
+		return true
+	}
+
+	if core.IsSmartContractAddress(scr.GetRcvAddr()) {
```

Comment:
Maybe this if could be put above L465

Reply:
nope. because you can have cross shard built in function call to a user